docs/search/solr_search_engine.md

```diff
@@ -29,10 +29,14 @@ Copy the necessary configuration files. In the example below from the root of yo
 # Make sure to replace the /opt/solr/ path with where you have placed Solr
 cd /opt/solr
 mkdir -p server/ez/template
-cp -R ./vendor/ibexa/solr/src/lib/Resources/config/solr/* server/ez/template
+cp -R <project_root>/vendor/ibexa/solr/src/lib/Resources/config/solr/* server/ez/template
 cp server/solr/configsets/_default/conf/{solrconfig.xml,stopwords.txt,synonyms.txt} server/ez/template
 cp server/solr/solr.xml server/ez
 
+# If you are using Ibexa Commerce, additionally copy commerce-specific configuration files:
+cat <project_root>/vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/custom-fields-types.xml >> server/ez/template/custom-fields-types.xml
+cat <project_root>/vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/language-fieldtypes.xml >> server/ez/template/language-fieldtypes.xml
+
```

Comment:
Hey @adamwojs do you think we can safely change `server/ez` into `server/ibexa`?
After a brief look, it would just require to align `generate-solr-config.sh` help text
https://github.com/ibexa/solr/blob/v4.2.0/bin/generate-solr-config.sh#L27

Reply:
It's find as long as we replace all `ez` references with `ibexa` (including this document)